README.rst

```diff
@@ -34,8 +34,8 @@ If you want a consistent emoji style, you can set it in your ``conf.py`` file:
 
    sphinxemoji_style = 'twemoji'
 
-You can find a list of all supported emoji codes in `the project's documentation page
```

Comment:
I would rather have descriptive links. In example, I prefer "the project's documentation page" rather than "here". It is descriptive and more explicit.

Reply:
In the new version of the documentation, I'm including the README inside the index.rst. That's why I've changed the wording. When the link is rendered on GitHub or PyPI, the wording "the project's documentation page" makes sense, but when it's rendered as part of the docs, it reads strange.

I like more descriptive links as well, but in this case I compromised on "here" for the above reasons. Nevertheless, I'll change it to "the project's documentation page" and we can improve it later if we can think of something better.

Reply:
Change made in 2333d37

Reply:
Oh, I understand. I am not a huge fan of including the README in the documentation. I would rather have the smallest README file, that links to the official documentation somewhere else. And just keep the README as simple as possible. :blush: